tests/terraform/graph/variable_rendering/test_resources/dynamic_blocks_with_nested/main.tf

```diff
@@ -0,0 +1,21 @@
+resource "azuread_application" "bootstrap" {
+  name                       = "test"
+  type                       = "webapp/api"
+  group_membership_claims    = "All"
+
+  dynamic "required_resource_access" {
+    for_each = var.required_resource_access
+    content {
+      resource_app_id = required_resource_access.value.resource_app_id
+
+      dynamic "resource_access" {
+        for_each = required_resource_access.value.resource_access
+        content {
+          id   = resource_access.value.id
+          type = resource_access.value.type
```

Comment:
Can we add a test for breadcrumbs for this configuration as well?
refer `test_dynamic_blocks_breadcrumbs` in `test_renderer.py`